Eliminate dependency on intel-mpi-rt-mic which in turn brings in intel-mpi-rt.
based on original patch by Bart Oldeman (McGill HPC)
--- pset/mediaconfig.xml.orig	2017-10-19 13:44:49.383480127 +0200
+++ pset/mediaconfig.xml	2017-10-19 13:48:09.237258398 +0200
@@ -888,7 +888,7 @@
     </Layers>
     <InstallDir>${COMPLIB_ROOT}</InstallDir>
     <GUID>1503574163037</GUID>
-    <Component depend="35.3;1.1;1.3;1.91;1.11;1.12;1.14;1.16;1.7;1.27;1.24;142.2" id="1" invisible="0" mandatory="0" platform="INTEL64">
+    <Component depend="35.3;1.1;1.3;1.91;1.11;1.12;1.14;1.16;1.7;1.27;1.24" id="1" invisible="0" mandatory="0" platform="INTEL64">
       <Name>Intel Fortran Compiler for Intel(R) 64</Name>
       <Description>Intel Fortran Compiler 18.0</Description>
       <Name locale="">Intel Fortran Compiler for Intel(R) 64</Name>
